py/picca/fitter2/effective-bins.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+import h5py
+#from picca.fitter2 import chi2,data,xi
+from picca.fitter2 import parser as fit_parser
```

Comment:
picca internal library go at the end of import list, easier to track external and internal library

Reply:
I don't understand